utils/preferences/src/extensions/unicode/keywords/emoji.rs

```diff
@@ -0,0 +1,11 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::enum_keyword;
+
+enum_keyword!(Emoji {
```

Comment:
Nit: Call it by its full name, `EmojiPresentationStyle`

Also elsewhere. I think we should use the full names because a lot of these concepts are ambiguous. An enum named "emoji" could be misinterpreted to mean a whole lot of other things.